### DISCUSSION TITLE
How do I run the Battery example?

### DISCUSSION THREAD

**Discussion by Robert-Adelard** [ge-high-assurance/RACK#460] [Q&A] [answered]
I've been trying to run the SemTk Battery example on the RACK - I can't use the SemTk demo server because I get 403 Forbidden.

https://github.com/ge-semtk/semtk/wiki/hello-world-sparqlgraph

I am having trouble setting up the connection and configuring the server.

In principle, I would expect `http://rack.server.com:3030` to work with `fuseki`, but I get error messages about a missing data endpoint server. However, there is no obvious way of specifying the data server.

How can I get this to work?

Thanks.

**Comment by cuddihyge** [comment, marked as answer]
Fuseki requires the dataset name on the end of the URL.   See the SemTK wiki page: [Triplestores](https://github.com/ge-semtk/semtk/wiki/Triplestores#apache-fuseki)

So ours are typically **http://localhost:3030/RACK**

You may look at the fuseki docs and it will show you how to add additional datasets to your fuseki instance if you don't want to use RACK.  
Use http://localhost:3030 and hit "manage datasets" at the top.    Then "add new dataset" is a little bit hidden in the gray bar.

**Reply by Robert-Adelard** [reply]
Thanks - it turns out that "add new dataset" is `Forbidden` for some reason, so it is only possible to use the RACK as the dataset.
